bin/express

```diff
@@ -134,6 +142,21 @@ function createApplication(app_name, path) {
       complete();
     });
 
+    // Index route
+    if (program.static) {
+      // set index route body
+      index = index.replace(
+        '{indexRouteBody}',
+        "res.sendfile('public/index.html');\n"
```

Comment:
What is the purpose of having a route with sendFile? What logic would someone add to the route that cannot output anything? Wouldn't it be better to leave off the route all together and let express.static connect index.html to the / request? I believe this is what various Angular boilerplates do.